docs/src/apimanual.md

```diff
@@ -174,7 +174,26 @@ addconstraint!(model, SingleVariable(x[1]), GreaterThan(0.0))
 addconstraint!(model, SingleVariable(x[2]), GreaterThan(-1.0))
 ```
 
-[Example with vector-valued set.]
+Besides scalar valued functions in scalar valued sets it possible to use vector valued functions and sets.
+
+The code example below encodes the linear optimization problem:
```

Comment:
It may be confusing to call this a linear optimization problem because it has a nonlinear (convex) constraint.